wiki/Tournaments/GBC/GBC_2023_Spring/en.md

```diff
@@ -0,0 +1,207 @@
+   - The total ranking of the mappool is based on the sum of the rankings of each player in each map.
+2. Beatmap scoring is based on Score V2.
+3. The GB Cup 2023 Spring consists of **4 groups (EX, S, A, and B)**, with only **2 groups** playing each week.
+   - Each group has an overrun buffer of *100 pp*.
+   - If a player upgrades after the start of the tournament, 50% of the points will be kept.
+4. The maps for each round will be announced by the mappool selectors on the official stream, on the Sunday before the actual matches take place.
+   - Each mappool contains 7 beatmaps of varying types.
+5. Players schedule their match time before the start of the week's match.
+   - Players with special time requirements need to contact the referee in advance. The organiser does not guarantee that anyone's request will be met, and players need to properly communicate with the referee in advance.
+6. Every week, the referee creates a multiplayer room at a fixed time, and the players who need to play a match will enter the room.
+7. There is **no warm-up phase** for the match.
+8. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
+   - Custom skin elements **must not** be used to alter core gameplay mechanics.
+9. Possible mod choices for the FreeMod bracket are **No Fail, **Hidden**, **Fade In**, **Flashlight**, and **Mirror**.
+10. The referees will select the beatmaps in order to play the match. If a player encounters a technical problem during a match, players may request a rematch from the referee after all the beatmaps have been selected.
+    - "Lag spikes" will not be considered as a valid reason to rematch a beatmap.
+    - The rematch should happen after completing scores for the other beatmaps in the mappool.
+    - This rule should not to be abused. Referees may veto a rematch request if they find that this is the case.
+11. Unless there are special reasons, the organiser will only record each player's first score of the beatmap in the match room.
+12. The default rest time is **120 seconds**. If all players in the room agree to start early and get ready, the referee may start the match early.
+13. Players are expected to keep the match running fluently and without delays. Excessive delays from the players' side may result in penalties being applied at the referee's discretion. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
+14. All players and staff must be treated with respect. Instructions from referees and the host are to be followed. Decisions labelled as final are not to be objected.
+15. The multiplayer chat rooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chat rooms, too.
```

Comment:
the second sentence doesn't make any sense. what is this intended to mean?

Reply:
Repeated sentences.
Deleted.